### ISSUE TITLE
Vancouver

### ISSUE THREAD

**Issue by saem** [papers-we-love/organizers#10]
tl;dr long time reading group, would like to raise Vancouver's profile

So we've (mostly me) been running a [reading group])(http://www.meetup.com/Polyglot-Vancouver-Reading-Group-Papers-We-Love-YVR/) in Vancouver for years now -- the name change was very recent. as an adjunct to another much larger [meetup](http://www.meetup.com/PolyglotVancouver/).

I know @tavisrudd, and @j14159 spoke to @zeeshanlakhani when ICFP was in Vancouver, and the whole joint meetup with Polyglot (I unfortunately couldn't be there as I was out of town). The impression I was given was rebranding was cool. Happened to swing by the site, and noticed we weren't mentioned on the chapters page. Found there is a whole process. Sorry about that, I'd like to fix that if we can.

We already have a [Code of Conduct](http://www.meetup.com/Polyglot-Vancouver-Reading-Group-Papers-We-Love-YVR/about/) (one we've used for both the meetups, and the annual Unconference we run). They read fairly similarly, so if there is an issue there, let me know.

**Comment by DarrenN** [comment]
Hello @saem - I'm a little confused as to what you want to do. Are you wanting to "re-brand" your Polyglot meetup into a Papers We Love Chapter?

**Comment by saem** [comment]
Hi @DarrenN, I'm hoping to keep both brands.

Polyglot is recognizable to locals (due to the meetup, and yearly unconference), so for meetup users it works great. There are lots of new faces coming to Vancouver and many come in that way.

Lots of people are not meetup users, but do use twitter and other avenues where PWL has bigger reach, and I'm hoping to come across their radar.

In short, I'd like as many people as possible in Vancouver to know, that if they feel they're missing out on a paper reading group, one exists, and they're most welcome.

**Comment by saem** [comment]
Excuse the rather protracted nature of this request, but there is some information missing based on your README that you might not have as of yet (explicitly at least), figured I'd add it here:

> About me:

Saem Ghani, CTO by day, Organizer of Polyglot Vancouver, Polyglot Unconference, and DevOpsDays YVR by night.

> Organizers

Saem Ghani https://twitter.com/saemg
Tavis Rudd https://twitter.com/tavisrudd
